docs/README.md

```diff
@@ -1,18 +1,3 @@
-Table of Contents
+## Read the generator docs at https://www.asyncapi.com/docs/tools/generator
 
-- [Introduction](index.md)
-- [Installation Guide](installation-guide.md)
-- [Usage](usage.md)
-- [AsyncAPI document](asyncapi-document.md)
-- [Template](template.md)
-- [Parser](parser.md)
-- [Generator version vs template version](versioning.md)
-- [Template Development](template-development.md)
-- [Configuration File](configuration-file.md)
-- [Template context](template-context.md)
-- [React render engine](react-render-engine.md)
-- [Nunjucks render engine](nunjucks-render-engine.md)
-- [Hooks](hooks.md)
-- [File templates](file-templates.md)
-- [TypeScript support](typescript-support.md)
-- [Special file names](special-file-names.md)
+Read the docs on asyncapi.com. You can still read the docs from here, but some of the links may be broken.
```

Comment:
## Documentation

Read the docs on https://www.asyncapi.com/docs/tools/generator. You can still read the docs from here, but some of the links may be broken.

Reply:
I think we should remove `Read the generator docs at https://www.asyncapi.com/docs/tools/generator` as it seems repeated (question mark!)